server/data/data-jmap/src/main/java/org/apache/james/jmap/api/vacation/Vacation.java

```diff
@@ -58,7 +59,7 @@ public Builder toDate(Optional<ZonedDateTime> toDate) {
             return this;
         }
 
-        public Builder textBody(String textBody) {
+        public Builder textBody(Optional<String> textBody) {
```

Comment:
you should also provide the same method with `String textBody` to easy the call when we know we have some content.